scripts/train/train.py

```
@@ -527,6 +527,24 @@ def main(cfg: DictConfig) -> Trainer:
         compile_config=compile_config,
     )
 
+    # Optionally just save an HF checkpoint
+    if train_cfg.only_hf_checkpoint:
+        hf_checkpointer_callbacks = [
+            c for c in callbacks if isinstance(c, HuggingFaceCheckpointer)
+        ]
+        if len(hf_checkpointer_callbacks) == 0:
+            raise ValueError(
+                'No HuggingFaceCheckpointer callback found, but only_hf_checkpoint was set to True. Please add a HuggingFaceCheckpointer.',
+            )
+        if len(hf_checkpointer_callbacks) > 1:
+            warnings.warn(
+                'Multiple HuggingFaceCheckpointer callbacks found. Only the first one will be used.',
+            )
```

i would just raise an error

---

sure